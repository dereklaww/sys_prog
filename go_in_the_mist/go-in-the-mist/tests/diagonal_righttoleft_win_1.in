place O19
place O1
place P18
place P1
place Q17
place Q1
place R16
place R1
place S15