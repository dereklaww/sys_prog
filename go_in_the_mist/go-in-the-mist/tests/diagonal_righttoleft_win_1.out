Black wins!
O19O1P18P1Q17Q1R16R1S15
Thank you for playing!